portals/admin/src/main/webapp/source/src/app/components/APISettings/ApisTableContent.jsx

```diff
@@ -0,0 +1,248 @@
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import React, { useState } from 'react';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import ResourceNotFound from 'AppComponents/Base/Errors/ResourceNotFound';
+import PropTypes from 'prop-types';
+import API from 'AppData/api';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Alert from 'AppComponents/Shared/Alert';
+import TextField from '@material-ui/core/TextField';
+import EditIcon from '@material-ui/icons/Edit';
+import { FormattedMessage } from 'react-intl';
+import { IconButton } from '@material-ui/core';
+import SaveIcon from '@material-ui/icons/Save';
+import CancelIcon from '@material-ui/icons/Cancel';
+
+const useStyles = makeStyles((theme) => ({
+    fullHeight: {
+        height: '100%',
+    },
+    tableRow: {
+        height: theme.spacing(5),
+        '& td': {
+            padding: theme.spacing(0.5),
+        },
+    },
+    appOwner: {
+        pointerEvents: 'none',
+    },
+    appName: {
+        '& a': {
+            color: '#1b9ec7 !important',
+        },
+    },
+    button: {
+        marginLeft: theme.spacing(0.5),
+        marginRight: theme.spacing(0.5),
+        marginTop: theme.spacing(1),
+    },
+    appTablePaper: {
+        '& table tr td': {
+            paddingLeft: theme.spacing(1),
+        },
+        '& table tr td:first-child, & table tr th:first-child': {
+            paddingLeft: theme.spacing(2),
+        },
+        '& table tr td button.Mui-disabled span.material-icons': {
+            color: theme.palette.action.disabled,
+        },
+    },
+    tableCellWrapper: {
+        '& td': {
+            'word-break': 'break-all',
+            'white-space': 'normal',
+        },
+    },
+    tableActionBtnContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'left',
+        justifyContent: 'left',
+        marginLeft: 10,
+    },
+    textfield: {
+        maxHeight: '10px',
+        marginTop: '-5px',
+        maxWidth: '120px',
+    },
+    tableCell: {
+        marginLeft: 10,
+    },
+}));
+
+const StyledTableCell = withStyles((theme) => ({
+    head: {
+        backgroundColor: theme.palette.common.black,
+        color: theme.palette.common.white,
+    },
+    body: {
+        fontSize: 14,
+    },
+    root: {
+        '&:first-child': {
+            paddingLeft: theme.spacing(2),
+        },
+    },
+}))(TableCell);
+
+const StyledTableRow = withStyles((theme) => ({
+    root: {
+        '&:nth-of-type(odd)': {
+            backgroundColor: theme.palette.background.default,
+        },
+    },
+}))(TableRow);
+
+const ApisTableContent = ({ apis, updateApiList }) => {
+    const [notFound] = useState(false);
```

Comment:
Do we need this state? If the state is not getting updated in the code we can remove it.

Reply:
https://github.com/wso2/apim-apps/pull/519/commits/426260ecaebe104033a2bdebe2ec8a03eef3d248